core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -71,20 +74,24 @@ public HadoopTables(Configuration conf) {
    */
   @Override
   public Table load(String location) {
+    Table result;
     Pair<String, MetadataTableType> parsedMetadataType = parseMetadataType(location);
 
     if (parsedMetadataType != null) {
       // Load a metadata table
-      return loadMetadataTable(parsedMetadataType.first(), parsedMetadataType.second());
+      result = loadMetadataTable(parsedMetadataType.first(), parsedMetadataType.second());
     } else {
       // Load a normal table
       TableOperations ops = newTableOps(location);
       if (ops.current() != null) {
-        return new BaseTable(ops, location);
+        result = new BaseTable(ops, location);
       } else {
         throw new NoSuchTableException("Table does not exist at location: " + location);
       }
     }
+
+    LOG.info("Table loaded by HadoopTables: {}", result);
```

Comment:
`HadoopTables` is logging this message, so there is no need to add it here.

Reply:
Done